main.py

```
@@ -142,7 +156,15 @@ def api_template(my_path=''):
         if UPLOAD_LIMIT is not None and CURRENT_UPLOAD_SIZE >= UPLOAD_LIMIT:
             return Response(status=400, response='Reached upload limit')
         with open(os.path.join('static/data/', my_path,  'template.json'), "w+") as outfile:
+            data = request.get_json()
             json.dump(request.get_json(), outfile, sort_keys=True, indent=4)
+        # Open template file to make sure what we just created matches what was submitted.
+        template = open(os.path.join('static/data/',(my_path + 'template.json')))
+        # Convert the template json to something validate can understand.
+        template = json.load(template)
+        # Call the validate function from jsonschema to verify our output json conforms with the template we were supposed to create
+        validate(data, template)
```

what happens if this fails? should return a descriptive error